Automattic/wp-calypso
# WIP: Animate header logo on processing screen in signup

#### Changes proposed in this Pull Request

* This is based on PR #28947. 
* This is an alternative approach to #28958. This animates the logo in the header from small to large on the processing screen. It also removes the duplicated logo from the processing screen.

![animatedlogo](https://user-images.githubusercontent.com/2124984/49253991-079c7a80-f3f6-11e8-8151-d453be003d09.gif)

(The GIF makes the animation/transitions look janky; they're much smoother in the browser :))

#### Testing instructions

* Switch to this PR, go to `/start`
* Go through the signup process
* Check for the animation on the processing screen after plan selection
